app/helpers/concerns/foreman_bootdisk/hosts_helper_ext.rb

```diff
@@ -6,20 +6,34 @@ module ForemanBootdisk::HostsHelperExt
   end
 
   def host_title_actions_with_bootdisk(*args)
+    if @host.architecture.blank? || @host.bootdisk_downloadable?
```

Comment:
I think accepting `architecture.blank?` is OK, but shouldn't this be in `bootdisk_downloadable?`, otherwise it will indicate false when it _is_ permitted.